### ISSUE TITLE
About GraphEngine  Data Loading

### ISSUE THREAD

**Issue by simple221** [microsoft/GraphEngine#293]
I need to store 600G of data, but only 32G of computer memory, the program will prompt memory overflow, how to solve, what solutions?

**Comment by TaviTruman** [comment]
@simple221 The GE is an In-Memory Graph Engine so if you want to load 600GB resident in RAM you will ultimately need available memory to load your data. The GE is a distributed memory cloud so you can load the data across what it "AvailabilityGroup" facility. Here is an excerpt from the GE Developer's Guide:

https://github.com/Microsoft/GraphEngine/wiki/config-v2

![image](https://user-images.githubusercontent.com/5692812/65543479-12b12180-dec6-11e9-8a1e-557c861f8312.png)

The GE also allows you to configure "Storage" nodes; you are going to want to experience with "TrunkCount" and "StorageCapacity" attributes.

**Comment by simple221** [comment]
> @simple221 The GE is an In-Memory Graph Engine so if you want to load 600GB resident in RAM you will ultimately need available memory to load your data. The GE is a distributed memory cloud so you can load the data across what it "AvailabilityGroup" facility. Here is an excerpt from the GE Developer's Guide:
> 
> https://github.com/Microsoft/GraphEngine/wiki/config-v2
> 
> ![image](https://user-images.githubusercontent.com/5692812/65543479-12b12180-dec6-11e9-8a1e-557c861f8312.png)
> 
> The GE also allows you to configure "Storage" nodes; you are going to want to experience with "TrunkCount" and "StorageCapacity" attributes.
okay,thank you.but i need stand-alone support.So maybe there's no solve.

**Comment by TaviTruman** [comment]
@simple221 Yes, if you need to run "stand-alone" on a single machine then and need to have all 600 GB loaded on then you would need to have available RAM on it. Unlike a lot of other Graph Engines that GE is an in-memory cloud; it does have a persistent storage capability so you can write to disk what you load into RAM so as to save it.

**Comment by simple221** [comment]
@TaviTruman  thank you.I see now.

**Comment by TaviTruman** [comment]
@simple221  if you're good with the explanation here you can close this issue.
